### PR TITLE
s4589325 OAI AKOA Perceiver

### PR DESCRIPTION
**Pull request by ghost**
Hi,

Please find my implementation of the Perceiver task for the OAI OKOA knee dataset. The final accuracy achieved was 93%. Details can be found in my README file.

Files:
- Driver_Script.py - driver script
- OAI_AKOA_Perceiver.py - Perceiver class definition
- Parameters.py - adjustable parameters (including hyperparameters)
- Slurp.sh - for running with CUDA
- README file
- Some related pictures/screenshots

Regards,
Ahmed Hafeez-Baig
s4589325

### CODE REVIEW

**Comment by SiyuLiu0329** [comment]
Difficulty Hard

Functionality: OK

Code
- Structure:  OK 

README 
- Content: Missing references to the paper and the guild (if any) you have used
-  Describe a bit more about the data (size etc?)
- Did you report test set acc, I can only see train and val

PR Target OK

General Comments
- Commit messages:  OK

**Comment by ghost** [comment]
Hi, I am not sure about the process for updating the project after the submission date (there is no response to the relevant question on EdStem), so I have committed the relevant changes to the README.

Thanks,
Ahmed

**Comment by shakes76** [comment]
## TF/Torch Usage
Adequate use and implementation
## Good Practice (Design/Commenting)
Good spacing and comments
Some Header blocks missing -1
## Algorithm 
Driver Script present
Shows Usage & Demo & Visualisation & Data usage 
Module present
No Data leakage
Difficulty: Hard
## Commit Log
Meaningful commit messages
Progressive commits used
## Documentation
Good Description and Comments 
Markdown used PDF submitted
## Pull Request
Successful Pull Request (Working Algorithm Delivered on Time in Correct Branch) 
No Feedback required
Request Description good
